setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name='django-markwhat',
-    version=".".join(map(str, __import__('django_markwhat').__version__)),
```

Comment:
Ok, no worries.
I'll fix the version number in the master after merging this pull-request.

Is this patch requires further changes? if not I'll merge the.

On Fri, Apr 3, 2015 at 11:09 PM, nikolas <notifications@github.com> wrote:

> In setup.py
> <https://github.com/Alir3z4/django-markwhat/pull/14#discussion_r27748959>:
>
> > @@ -2,7 +2,7 @@
> >
> >  setup(
> >      name='django-markwhat',
> > -    version=".".join(map(str, __import__('django_markwhat').__version__)),
>
> I hardcoded the version number here because this line was giving me an
> error when I tried to run tox on my checkout.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Alir3z4/django-markwhat/pull/14/files#r27748959>.
>